scripts/skills/perks/perk_rf_second_wind.nut

```diff
@@ -0,0 +1,32 @@
+this.perk_rf_second_wind <- ::inherit("scripts/skills/skill", {
+	m = {
+		ActionPointsTarget = 4	// set the current Action Points to this value if it's lower
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_second_wind";
+		this.m.Name = ::Const.Strings.PerkName.RF_SecondWind;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_SecondWind;
+		this.m.Icon = "ui/perks/rf_second_wind.png";
+		this.m.Overlay = "rf_second_wind_effect";
```

Comment:
1. We use `_effect` for brush icons only for actual status effects. For perks we use the same name as the perk's icon with `perk_` prefixed to it. So the sprite's name in this case should be `perk_rf_second_wind`. As a guideline, keep it the same as the filename of the skill.
2. Do not add this as `this.m.Overlay` because `this.m.Overlay` is triggered when a skill is added to an entity, or when an active skill is used by an entity. We don't want the icon to automatically spawn if e.g. this perk is added to an entity during battle somehow.